packages/eslint-config-godaddy/package.json

```diff
@@ -3,37 +3,39 @@
   "version": "7.1.1",
   "description": "Base ESLint config for consistent style in ES6 projects at GoDaddy.",
   "license": "MIT",
+  "type": "module",
+  "main": "index.js",
   "scripts": {
     "lint": "eslint .",
     "pretest": "npm run --silent lint",
     "test": "echo ok",
-    "clean": "rimraf node_modules",
-    "unused": "eslint-find-rules --unused ./index.js || echo ''"
+    "clean": "rimraf node_modules"
   },
   "repository": {
     "type": "git",
-    "url": "git@github.com:godaddy/javascript.git"
+    "url": "git@github.com:godaddy/javascript.git",
```

Comment:
Side note... we really ought to rename this repo.

Reply:
Agreed though naming is hard sometimes 😅